### PR TITLE
PHP: Fixed issue #5375 for intersection type

### PR DESCRIPTION
**Pull request by troizet**

### CODE REVIEW

File: php/php.editor/test/unit/data/testfiles/completion/documentation/issueGH5375Intersection.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public ClassA&ClassB $test_with_var_tag;
+
+    public function test() {
+        $this->test_without_var_tag;
+        $this->test_with_var_tag;
+    }
+
+}
+
+?>
```
Comment (junichi11) [inline php/php.editor/test/unit/data/testfiles/completion/documentation/issueGH5375Intersection.php:45]: Nitpick: Need not `?>`

**Review by junichi11 — APPROVED** [review]
Thanks!

**Review by tmysik — APPROVED** [review]
Thanks!

File: php/php.editor/test/unit/data/testfiles/completion/documentation/issueGH5375Intersection.php (hunk 1 above)
Comment (troizet) [reply php/php.editor/test/unit/data/testfiles/completion/documentation/issueGH5375Intersection.php]: Fixed.

**Comment by troizet** [comment]
Thanks!

**Pull request merged**